src/test/java/domain/DuplicateLottoNumbersRemoverTest.java

```diff
@@ -0,0 +1,47 @@
+package domain;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class DuplicateLottoNumbersRemoverTest {
+    @Test
+    @DisplayName("LottoNumbers 중복 제거 테스트")
+    void removeDuplicatesTest() {
+        // given
+        List<LottoNumbers> manualLottoNumbersGroup = new ArrayList<>();
+        manualLottoNumbersGroup.add(new LottoNumbers(Stream.of(1, 2, 3, 4, 5, 6)
+                .map(LottoNumber::new)
+                .collect(Collectors.toList())
+        ));
+        manualLottoNumbersGroup.add(new LottoNumbers(Stream.of(1, 2, 3, 4, 5, 7)
+                .map(LottoNumber::new)
+                .collect(Collectors.toList())
+        ));
+
+        List<LottoNumbers> autoLottoNumbersGroup = new ArrayList<>();
+        autoLottoNumbersGroup.add(new LottoNumbers(Stream.of(1, 2, 3, 4, 5, 6)
+                .map(LottoNumber::new)
+                .collect(Collectors.toList())
+        ));
+        autoLottoNumbersGroup.add(new LottoNumbers(Stream.of(1, 2, 3, 7, 8, 9)
+                .map(LottoNumber::new)
+                .collect(Collectors.toList())
+        ));
+        // when
+        DuplicateLottoNumbersRemover.removeDuplicates(manualLottoNumbersGroup, autoLottoNumbersGroup);
+        // then
+        LottoNumbers lottoNumbers = new LottoNumbers(Stream.of(1, 2, 3, 4, 5, 6)
+                .map(LottoNumber::new)
+                .collect(Collectors.toList())
+        );
```

Comment:
각 테이터 만들어주는 부분들을 메서드로 분리하면 가독성이 더 좋을 것 같아요~
```suggestion
        List<LottoNumbers> manualLottoNumbersGroup = 메서드명();
        List<LottoNumbers> autoLottoNumbersGroup = 메서드명();
        
        // when
        DuplicateLottoNumbersRemover.removeDuplicates(manualLottoNumbersGroup, autoLottoNumbersGroup);
        
        // then
        LottoNumbers lottoNumbers = new LottoNumbers(Stream.of(1, 2, 3, 4, 5, 6)
                .map(LottoNumber::new)
                .collect(Collectors.toList())
        );
```

Reply:
테스트 코드에서도 메소드 분리로 가독성을 개선할 수 있다는 점을 망각했네요..
메소드로 분리해서 개선해보겠습니다.